mobile/src/components/interviews/interviews-table/components/id-cell/id-cell.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+
+import { InterviewsUpdateRequestParamsDto } from '~/common/types/types';
+import { Pressable, Text } from '~/components/common/common';
+
+import { styles } from './styles';
+
+type Props = {
+  id: number;
+  onPress: (id: InterviewsUpdateRequestParamsDto) => void;
+};
+
+const IdCell: FC<Props> = ({ id, onPress }) => {
+  return (
+    <Pressable onPress={(): void => onPress({ id })}>
```

Comment:
Move this outside jsx
```suggestion
    <Pressable onPress={handlePress}>
```